cypress/integration/QuerryParamsRedirects.js

```diff
@@ -0,0 +1,24 @@
+// @noflow
+/* global cy */
+
+describe('Querry params redirect to corect article', () => {
+  it(`User should see article about cabin baggage`, () => {
+    cy.visit(
+      '/?cypress-query=/faq/RkFRQ2F0ZWdvcnk6ODk=/article/RkFRQXJ0aWNsZTo0MQ==',
+    );
+    cy.get('[data-cy=faq-breadcrumbs]').as('faq-breadcrumbs');
+    cy.get('[data-cy=faq-article-form]').as('faq-article-form');
+
+    cy
```

Comment:
I would suggest to use `it('...')` to describe that 1) there are breadcrumbs 2) form 3) article with such title

Reply:
thanks for the suggestion! 😊